src/benchcab/benchcab.py

```diff
@@ -245,13 +245,14 @@ def fluxsite_submit_job(self, config_path: str, skip: list[str]) -> None:
         logger.info("The NetCDF output for each task is written to:")
         logger.info(f"{internal.FLUXSITE_DIRS['OUTPUT']}/<task_name>_out.nc")
 
-        # Upload to meorg by default
-        bm.do_meorg(
-            config,
-            upload_dir=internal.FLUXSITE_DIRS["OUTPUT"],
-            benchcab_bin=str(self.benchcab_exe_path),
-            benchcab_job_id=job_id,
-        )
+        # Upload to meorg if meorg_output_name optional key is passed
+        if config.get("meorg_output_name") is not None:
```

Comment:
`meorg_output_name` is not a key in the top level config

```suggestion
        if any(r.get("meorg_output_name" for r in config["realisations"]):
```

Reply:
While processing `config.yaml`, after choosing the realisation which has `meorg_output_name` as `True`,  the realisation name is derived and set to top level for `config`

https://github.com/CABLE-LSM/benchcab/blob/45e1456a0f5afd1c30971169ee8cfd9e9ad2e591/src/benchcab/config.py#L206